admin/services/data-access/pupil-access-arrangements.data.service.js

```diff
@@ -157,4 +157,25 @@ pupilAccessArrangementsDataService.sqlFindAccessArrangementsByUrlSlug = async (u
   return sqlService.query(sql, params)
 }
 
+/**
+ * Delete pupil's access arrangements.
+ * @param {String} urlSlug
+ * @return {Object}
+ */
+pupilAccessArrangementsDataService.sqlDeletePupilsAccessArrangements = async (urlSlug) => {
```

Comment:
Just a thought, should we soft-delete the access arrangements instead? 🤔

Reply:
@danwilliams is there any important reason for it?